content/cybersecurity/concepts/virtual-private-network/virtual-private-network.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: 'Virtual Private Network'
+Description: 'Virtual Private Network (VPN) is a solution to allow remote access for employees to connect to their organization's internal network from their home or while traveling.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Computer Science'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+**Virtual Private Network (VPN)** is a solution to allow remote access for employees to connect to their organization's internal network from their home or while traveling. VPN traffic goes over a public network so encryption is important. A VPN provides a secure communications method for data transmitted between networks.
+
+## Virtual Private Network In-Use
+
+Employees will often use a VPN when working remotely in order to connect to their company network. While public Wi-Fi is convenient, it is also insecure. Using a VPN ensures that sensitive company data is encrypted. This means that it works like a filter by turning all your data into indecipherable text.
+
+## Different Types of VPN
+
+There are three types of VPN:
+- Site-to-site VPN: Used to connect branch offices to a central office over the internet when the offices are separated by distance.
+- Remote access VPN: Used to allow individual users to remotely connect to a central network. The remote devices that connect to the network are referred to as endpoints.
+- Cloud VPN: Also known as a hosted VPN, it allows users to securely access a company’s applications, data and files in the cloud through a website, desktop or mobile application. Cloud VPNs are accessible and do not require any type of VPN infrastructure on the user’s end. In other words, a company can authenticate users without having to purchase expensive hardware or networking equipment.
```

Comment:
I'd also love to see some example of VPNs following this section

Reply:
I, unfortunately, don't have some examples of VPNs for this section. The contributor that worked on this content left.

Reply:
Alright, in which case I can make a note that this may be something to add for the next iteration's edits